features/hear_shout.feature

```diff
@@ -1,10 +1,12 @@
 Feature: Hear shout
   Scenario: Listener is within range
-    Given Lucy is located 15m from Sean
+    Given Sean is located at (0, 0)
```

Comment:
I'm not a big fan of having the locations directly written in the scenarios. I know it's how it's implemented in the original Shouty.js but I find it a big confusing and making the feature hard to read.

Reply:
Ok, I can try to change that